src/components/AppController/AppController.js

```diff
@@ -57,16 +64,21 @@ var App = React.createClass({
         <Row>
           <Col lg={6}>
             <div>
-              <MapController
+              <TheMapController
                 meta = {this.getfilteredMeta()}
+                area={this.state.area}
                 onSetArea={this.handleSetArea}
               />
             </div>
           </Col>
           <Col lg={6}>
             <DataController
-              // ensemble_name, model_id, variable_id, experiment, area
-              {...this.state}
+              ensemble_name={this.state.ensemble_name}
```

Comment:
Yeah, that's a problem throughout. I considered, as a general practice, pulling the required properties into a few separate objects, but that didn't seem to enhance readability much. 

It might just be worth it in MapController where there is so much passing of values into components as props, often the same or very similar groups of values to several different components.